projects/isy-angular-widgets/src/lib/hauptfenster/hauptfenster.component.html

```diff
@@ -83,15 +84,16 @@
             (click)="collapsedLinksnavigation = false"
           ></button>
         </div>
-      </div>
+      </aside>
     }
 
-    <div class="col overflow-auto">
+    <div role="main" aria-label="main" class="col overflow-auto">
```

Comment:
It is better to use the main tag here. role="main" and aria-label="main" can be deleted.